### ISSUE TITLE
servo panic on shutdown

### ISSUE THREAD

**Issue by metajack** [servo/servo#9828]
```
thread '<main>' panicked at 'called `Result::unwrap()` on an `Err` value: "SendError(..)"', ../src/libcore/result.rs:746
stack backtrace:
   1:     0x556c0dc5c330 - sys::backtrace::tracing::imp::write::hb39adb9cc1dae4a9Nyu
   2:     0x556c0dc606eb - panicking::default_handler::_$u7b$$u7b$closure$u7d$$u7d$::closure.43683
   3:     0x556c0dc60358 - panicking::default_handler::hced6c0d85eb2cb83bez
   4:     0x556c0dc49efc - sys_common::unwind::begin_unwind_inner::hafb014b2dc8828386mt
   5:     0x556c0dc4a388 - sys_common::unwind::begin_unwind_fmt::hd671b811c60b4ff4cmt
   6:     0x556c0dc5b811 - rust_begin_unwind
   7:     0x556c0dc90d8f - panicking::panic_fmt::h38cc775b27fedfe3yZL
   8:     0x556c0c2f1abe - result::unwrap_failed::h14530111616767767096
   9:     0x556c0c323b23 - compositor::IOCompositor<Window>::process_animations::h16356338416759592920
  10:     0x556c0c3226e9 - compositor::IOCompositor<Window>::composite_specific_target::h4308079088374098341
  11:     0x556c0c327f21 - compositor::IOCompositor<Window>::composite::h4429866135980220272
  12:     0x556c0c2f4d93 - compositor::IOCompositor<Window>.CompositorEventListener::handle_events::h2397621241285639775
  13:     0x556c0c32e19c - Browser::handle_events::h76bf5617ef72e836ifa
  14:     0x556c0c2def47 - main::h8517eb49d15e90fbNaa
  15:     0x556c0dc5ffb4 - sys_common::unwind::try::try_fn::h10253683436607047365
  16:     0x556c0dc5b79b - __rust_try
  17:     0x556c0dc5fa4b - rt::lang_start::h93379fbffda56b55h6y
  18:     0x7f05d551d57f - __libc_start_main
  19:     0x556c0c2d4b18 - _start
  20:                0x0 - <unknown>
```

Steps to reproduce (on linux):

1. `./mach run --release -- -i -w --resolution 1280x1024 https://github.com`
2. Hit escape to exit one logging stops
3. Repeat steps 1 and 2 until panic. Takes 3-4 times on average for me.

**Comment by metajack** [comment]
cc @glennw @pcwalton

**Comment by asajeffrey** [comment]
Potentially related to #10017.

**Comment by ConnorGBrewster** [comment]
Is this still an issue? I cannot reproduce.

**Comment by asajeffrey** [comment]
Fixed by #10082.
